src/test/scala/de/m7w3/signal/ApplicationContextTest.scala

```diff
@@ -0,0 +1,7 @@
+package de.m7w3.signal
+
+import de.m7w3.signal.Config.SignalDesktopConfig
+import scala.concurrent.duration._
+import java.io.File
+
+object ApplicationContextTest extends ApplicationContext(SignalDesktopConfig(false, 1.seconds, new File("foo")))
```

Comment:
would rather create a trait that extends beforeAndAfter... and contains an ApplicationContext pointing to some tempdir that is deleted afterAll